runtime/help/plugins.md

```diff
@@ -121,6 +121,17 @@ The packages and functions are listed below (in Go type signatures):
        current pane is not a BufPane.
 
     - `CurTab() *Tab`: returns the current tab.
+
+    - `Redraw()`: update the screen contents. In most cases micro itself
+       redraws the screen when needed, but in cases when it doesn't (e.g.
+       if a lua code is running asynchronously, e.g. in a timer callback)
+       this function can be used to explicitly request screen update.
+
+    - `Lock`: Access the Lua's lock mutex. This is useful to prevent 
+      simultaneous access to a shared resource from different threads.
+
+    - `Tabs() *TabList`: returns the list of all tabs (in `Tabs().List`)
```

Comment:
Nit: it's better to move it above `Redraw()`, to group it together with `CurTab()`.

Reply:
Ok, done.